tfrog-motordriver/controlVelocity.c

```diff
@@ -362,6 +369,7 @@ void controlVelocity_init()
   driver_param.admask = 0;
   driver_param.io_mask[0] = 0;
   driver_param.io_mask[1] = 0;
+  driver_param.control_cycle = 1;
```

Comment:
Initial value at boot time was random.
It could cause very long interval of velocity calculation; count/interval value was very large if motor is rotating at start time.